### DISCUSSION TITLE
ORM like features.

### DISCUSSION THREAD

**Discussion by kastriotkastrati** [tinyplex/tinybase#20] [Ideas]
Hello, I want to know your opinions on minimal ORM like features like "limit" functions. 

I know the project is very focused towards small bundle sizes, so this and other similar functionalities fall upon us the community to make in a separate tinybase utils package when the time comes.

I've recently been accessing a lot of data via tinybase and at times, when I have needed to paginate content for example, I have felt that such minimal functions would have been nice.

**Comment by jamesgpearce** [comment]
Hopefully you saw there are sorting and limit features in v2.0 (both against raw Store tables as well as query results). As for other ORM ideas (like code-genned method names based on schemas... stay tuned to the beta branch!

**Comment by jamesgpearce** [comment]
Take a look at https://beta.tinybase.org/guides/developer-tools/command-line/ - and about to roll to final release
